packages/devices/origin-device-registry-irec-local-api/src/device/device.controller.ts

```diff
@@ -88,7 +88,7 @@ export class DeviceController {
         description: 'Returns all IREC fuels types'
     })
     getFuelTypes(): CodeNameDTO[] {
```

Comment:
These controller functions (+ descriptions) should be renamed as well, because their naming is later transferred to auto-generated clients.